components/choose_download_action_dialog.py

```diff
@@ -0,0 +1,66 @@
+from PySide2.QtWidgets import QRadioButton
+from PySide.QtWidgets import QDialog, QDialogButtonBox, QVBoxLayout, QLabel
+import Utils
+from APIClient import ConnStatus
+
+logger = Utils.getLogger(__name__)
+
+
+class ChooseDownloadActionDialog(QDialog):
+    def __init__(self, name, data_parent, parent=None):
+        super().__init__(parent)
+        self.quit_on_close = False
+        conn_status = data_parent.api.getStatus()
+
+        self.setWindowTitle(name)
+        self.answer = 0  # 0 = cancel, 1 = open web, 2 = download to memory
```

Comment:
I would prefer to have constants here (they are actually already there in the comment)